MailContainerTest/Services/MailTransferService.cs

```diff
@@ -1,93 +1,42 @@
 ﻿using MailContainerTest.Data;
 using MailContainerTest.Types;
-using System.Configuration;
 
 namespace MailContainerTest.Services
 {
     public class MailTransferService : IMailTransferService
     {
-        public MakeMailTransferResult MakeMailTransfer(MakeMailTransferRequest request)
-        {
-            var dataStoreType = ConfigurationManager.AppSettings["DataStoreType"];
-
-            MailContainer mailContainer = null;
-
-            if (dataStoreType == "Backup")
-            {
-                var mailContainerDataStore = new BackupMailContainerDataStore();
-                mailContainer = mailContainerDataStore.GetMailContainer(request.SourceMailContainerNumber);
+        private readonly IMailContainerDataStore _containerDataStore;
+        private readonly IMakeMailTransferRequestApprovalService _requestApprovalService;
 
-            } else
-            {
-                var mailContainerDataStore = new MailContainerDataStore();
-                mailContainer = mailContainerDataStore.GetMailContainer(request.SourceMailContainerNumber);
-            }
+        public MailTransferService(
+            IMailContainerDataStore containerDataStore, 
```

Comment:
I would ensure these two dependencies had implementations registered in some IoC container, e.g. in a Startup.cs or Program.cs class for a web API project, before instantiating this class.